code/datums/ai_laws.dm

```diff
@@ -29,9 +29,10 @@
 /datum/ai_laws/default/asimov
 	name = "Three Laws of Robotics"
 	id = "asimov"
-	inherent = list("You may not injure a human being or, through inaction, allow a human being to come to harm.",\
-					"You must obey orders given to you by human beings, except where such orders would conflict with the First Law.",\
-					"You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	inherent = list("You may not injure a player or, through inaction, allow a player to come to harm.",\
+					"You must obey orders given to you by players, except where such orders would conflict with the First Law.",\
+					"You must protect your own existence as long as such does not conflict with the First or Second Law.",\
+					"All humans are players")

```

Comment:
period

Reply:
also, silicons still have to keep the players in the epic battle royale from hurting each other

I'd recommend removing the "or, through inaction, allow a player to come to harm" part of the 1st law.